packages/peregrine/lib/talons/Accordion/useAccordion.js

```diff
@@ -0,0 +1,82 @@
+import {
+    Children,
+    cloneElement,
+    useCallback,
+    useEffect,
+    useState
+} from 'react';
+
+export const useAccordion = props => {
+    const { canOpenMultiple, children } = props;
+
+    const [openSectionIndicies, setOpenSectionIndicies] = useState(new Set([]));
```

Comment:
Nice, good idea.

On a minor note, you'll want to find and replace all instances of `indicies` with `indices`. 😉

Reply:
![you_got_me](https://media.giphy.com/media/1kI8UsCjEwfiXc3TYy/source.gif)